presto-function-namespace-managers/src/main/java/com/facebook/presto/functionNamespace/AbstractSqlInvokedFunctionNamespaceManager.java

```diff
@@ -219,14 +219,20 @@ protected static FunctionMetadata sqlInvokedFunctionToMetadata(SqlInvokedFunctio
                         .collect(toImmutableList())),
                 function.getSignature().getReturnType(),
                 SCALAR,
-                function.getFunctionImplementationType(),
```

Comment:
Are we going to remove `getFunctionImplementationType` this method now that it does not have any callers?

Reply:
It is removed.